web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -503,6 +503,8 @@ $.extend(fixmystreet.set_up, {
         addRemoveLinks: true,
         thumbnailHeight: 150,
         thumbnailWidth: 150,
+        resizeWidth: 1600,
```

Comment:
I'm fine with this, but for some reason I have a niggling desire for it to be more like 2048 (3mp) than 1600. Can't really justify the feeling, how much difference do you think that would make to file size?